graph.html

```diff
@@ -8,6 +8,7 @@
     function drawChart() {
         var dataSource = new google.visualization.arrayToDataTable([
             ['Framework', 'Response', 'Graal'],
+            ["Avaje", avaje, GRAALAVAJE],

```

Comment:
```suggestion
            ["Avaje", AVAJE, GRAALAVAJE],
```
better to make it capital easier to catch this way like in quarkus.